src/Test/SymfonyTest.php

```diff
@@ -108,14 +108,21 @@ protected function getProxy()
      *
      * @return Symfony
      */
-    protected function getProxyClient()
+    protected function getProxyClient(array $options = array())
     {
+        $options = array_merge(
+            [
+                'purge_method' => 'NOTIFY'
```

Comment:
this was/is used to test whether the configuration option to change the purge_method works. but indeed not sure why we would need that in the Test - having it in one unit test would be enough. we should not fix this in this PR - its quite big already.